Gemfile

```diff
@@ -1,3 +1,11 @@
 source 'https://rubygems.org'
 
 gemspec
+
+gem 'rb-inotify', '~> 0.9', '>= 0.9.7'
+gem 'daemons', '~> 1.0', '>= 1.0.9'
+gem 'concurrent-ruby', '~> 1.1.3'
+# Bundler does not take current ruby version into account.
```

Comment:
Maybe say `required_ruby_version` instead of current?

Reply:
Also as discussed IRL, no need to provide a range. Just directly reference the patch version in `~>`. replace first parameter with the second. no need 2 params

Reply:
Hey @danielbreves ,

I discovered something about `daemons` so I had to go with
```
gem 'rb-inotify', '~> 0.9.7'
gem 'concurrent-ruby', '~> 1.1.3'
gem 'daemons', '>= 1.0.9', '<= 1.3.0'
```

because current we rely on `demons` version `1.0.9` and `1.3.0`

Reply:
😈